test/correlationId.test.ts

```diff
@@ -0,0 +1,81 @@
+import axios, { AxiosInstance } from "axios";
+import { randomUUID } from "crypto";
+import { DateTime } from "luxon";
+import { ConnectorClientWithMetadata, Launcher } from "./lib/Launcher";
+import { getTimeout } from "./lib/setTimeout";
+import { establishRelationship } from "./lib/testUtils";
+
+const launcher = new Launcher();
+let axiosClient: AxiosInstance;
+let connectorClient1: ConnectorClientWithMetadata;
+let connectorClient2: ConnectorClientWithMetadata;
+let account2Address: string;
+const uuidRegex = new RegExp("[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}");
+
+beforeAll(async () => {
+    [connectorClient1, connectorClient2] = await launcher.launch(2);
+    axiosClient = axios.create({
+        baseURL: connectorClient1._baseUrl,
+        validateStatus: (_) => true
+    });
```

Comment:
if you use the client from connectorClient1 instead you can leave out setting the api key below and you don't have to expose the baseurl:

```suggestion
    axiosClient = connectorClient1["account"]["httpClient"];
```

Reply:
Yes already in work but we need to add the ability in the SDK to add a custom correlation ID.

This will be a bigger change, so it will come in another PR